phoonnx/model_manager.py

```
@@ -396,10 +406,31 @@ def get_phonikud_voice_list(self):
                 lang="he",
                 model_url="https://huggingface.co/thewh1teagle/phonikud-tts-checkpoints/resolve/main/shaul.onnx",
                 config_url="https://huggingface.co/thewh1teagle/phonikud-tts-checkpoints/resolve/main/model.config.json",
-                phoneme_type=PhonemeType.PHONIKUD
+                phoneme_type=PhonemeType.PHONIKUD,
+                engine=Engine.PIPER
             )
         )
 
+    def get_mms_voice_list(self):
+        url = "https://huggingface.co/willwade/mms-tts-multilingual-models-onnx/raw/main/languages-supported.json"
+        for data in requests.get(url).json():
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)